index.d.ts

```diff
@@ -38,6 +38,17 @@ declare namespace execa {
 		*/
 		readonly localDir?: string;
 
+		/**
+		Path to the Node.js executable to use in child processes.
+
```

Comment:
Would be great to include the use-cases for this option.

Reply:
Done in 734450e6922e917ef5835a63b8edbaf47e43ce6c

Reply:
Done in 11029d47b183dbb625fe1af5a8d7037218d296b1